docs/getting-started/user-guides/agenda-voting.md

```diff
@@ -30,111 +30,70 @@ Below is a diagram of the entire cycle for a single agenda with consensus upgrad
+Finally, you'll need to learn how to set the `votechoice` for your tickets in order to cast a "Yes", "No", or "Abstain" vote for an agenda. By default, your tickets will cast "Abstain" votes. To set your vote choice, see our quick [How To Vote](/getting-started/user-guides/how-to-vote.md).
 
 ---
 
-## **Solo-voting**
-
-Through the command-line, you'll want to familiarize yourself with the `dcrctl --wallet getvotechoices` and `dcrctl --wallet setvotechoice "agendaid" "choiceid"` commands.
-
-The first command, `dcrctl --wallet getvotechoices`, returns JSON resembling this:
-
-```
-{
-  "version": 4,
-  "choices": [
-    {
-      "agendaid": "sdiffalgorithm",
-      "agendadescription": "Change stake difficulty algorithm as defined in DCP0001",
-      "choiceid": "abstain",
-      "choicedescription": "abstain voting for change"
-    },
-    {
-      "agendaid": "lnsupport",
-      "agendadescription": "Request developers begin work on Lightning Network (LN) integration",
-      "choiceid": "abstain",
-      "choicedescription": "abstain from voting"
-    }
-  ]
-}
-```
-
-The second command, `dcrctl --wallet setvotechoice "agendaid" "choiceid"`, let's you set your votechoice. `"agendaid"` is found via the `getvotechoices` command above, and `"choiceid"` can be `yes`, `no`, or `abstain`.
-
-For example, issuing `dcrctl --wallet setvotechoice sdiffalgorithm yes` results in the following changes to `dcrctl --wallet getvotechoices`:
-
-```
-{
-  "version": 4,
-  "choices": [
-    {
-      "agendaid": "sdiffalgorithm",
-      "agendadescription": "Change stake difficulty algorithm as defined in DCP0001",
-      "choiceid": "yes",
-      "choicedescription": "change to the new algorithm"
-    },
-    {
-      "agendaid": "lnsupport",
-      "agendadescription": "Request developers begin work on Lightning Network (LN) integration",
-      "choiceid": "abstain",
-      "choicedescription": "abstain from voting"
-    }
-  ]
-}
-```
-
----
+## **Verifying Your Vote**
 
-## **Stakepool-voting**
+The easiest method to track your how your tickets actually voted is to use the [block explorer](https://mainnet.decred.org) to view each of your votes.
 
-If your Stakepool has updated to the latest stakepool software, you will find a "Voting" page in the navigation menu with dropdown options for each agenda. After you've chosen how you want your tickets to vote, simply press the "Update Voting Preferences" to save your votechoices. Below you'll find an image of the votechoices for vote version 4.
+The block explorer has been updated to display YES, NO, and ABSTAIN votes for each agenda with each ticket. The first and second tickets in the following example image voted "ABSTAIN" for both agendas while the third ticket voted "YES" for both agendas.
```

Comment:
Should the `YES`, `NO`, and `ABSTAIN` words be quoted to match the existing style that quotes them everywhere?